Sources/FluidAudioCLI/Commands/VadBenchmark.swift

```diff
@@ -430,9 +437,11 @@ struct VadBenchmark {
                 let vadResults = try await vadManager.process(url)
                 inferenceTime += Date().timeIntervalSince(inferenceStartTime)
 
-                // Aggregate results (use max probability as file-level decision)
+                // Aggregate results (use activity ratio as file-level decision)
+                let activeChunks = vadResults.filter { $0.isVoiceActive }.count
+                let activityRatio = Float(activeChunks) / Float(vadResults.count)
+                let prediction = activityRatio >= activityThreshold ? 1 : 0
```

Comment:
I think having some thershold here is better than taking one point out of hundres and getting the max there, its a bit fleky..